vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/executor/listeners/pull/PullScreenshotTestRunListener.kt

```diff
@@ -83,10 +81,7 @@ class PullScreenshotTestRunListener(
 
     private fun removeScreenshots(applicationId: String) {
         val remoteDir = device.fileManager.getScreenshotsDir(applicationId)
-        val millis = measureTimeMillis {
-            device.fileManager.removeFromFilesDir(applicationId, remoteDir)
-        }
-        logger.trace { "Removed files from $remoteDir in ${millis}ms" }
```

Comment:
I had a look at VRT outputs and this is 1 line of log. I doubt it makes much of a difference.

I think a better candidate would be where we write the test name, i.e.
"Test(pkg=com.badoo.designdemo, clazz=VisualRegressionTest, method=test[134], metaProperties=[], componentInfo=AndroidComponentInfo(name=:design:DesignDemo:debug, applicationId=com.badoo.designdemo, testApplicationId=com.badoo.designdemo.test, applicationOutput=/opt/buildagent/work/cce15ccf84df2911/design/DesignDemo/build/outputs/apk/debug/DesignDemo-debug.apk, testApplicationOutput=/opt/buildagent/work/cce15ccf84df2911/design/DesignDemo/build/outputs/apk/androidTest/debug/DesignDemo-debug-androidTest.apk))"

That rubbish is logged numerous times, and it's always the same (except for the method name). Perhaps we could keep the method name and strip out everything else.

Reply:
I replaced logging of `Test` with test name where possible.